### DISCUSSION TITLE
Artisan stuck in Artisan viewer mode

### DISCUSSION THREAD

**Discussion by Million-Volt** [artisan-roaster-scope/artisan#865] [Q&A]
Help! How do you close the Artisan viewer? I had just started warming up for a roast, and clicked on an old .alog profile and it opened Artisan Viewer and I lost all my controls. 

I tried restarting the software and it still opens in artisan viewer. I've even tried deleting and reinstalling artisan to the current 2.6.0 version and it still only opens in artisan viewer.

I've trawled through forums and can't find much information on this. It's incredibly frustrating. Could be because I haven't had my coffee yet!?!

Running macOS 11.6.3
Hottop KN-8828B-2K+

Any advice would be much appreciated. Thanks!

Chris

**Comment by poundy** [comment]
Artisan Viewer will only open (on a PC) when you already have Artisan open.  So when you opend the old ALOG (by clicking it) with Artisan already open, it will just open it in viewer. You could have gone to the File-Open menu and opened the old file successfully within the same Artisan session. 

Sounds to me you have an Artisan session hiding behind some other window you need to get to  

Edit: I missed you're on a Mac. You have to go thru a specific copy process to get a second instance on a Mac. Perhaps you need to find the original application and open it, and it looks like the default is to open the Viewer copy...

**Reply by Million-Volt** [reply]
Thanks for your reply poundy. Yes, that sounds like a rational reason for my issue. 

Unfortunately I cannot find the original Artisan window. Only the Artisan viewer displays with all its limited functionality. 

Thanks, but still looking for a solution...

**Reply by MAKOMO** [reply]
Use the task manager to terminate the hanging Artisan process or just restart your PC.

**Reply by Million-Volt** [reply]
(slaps palm to face) Thanks Makomo! That fixed it.

Seems strange that reinstalling the software wouldn't achieve the same thing.

Thanks again everyone!
